scripts/build_avado.sh

```diff
@@ -81,5 +88,7 @@ sudo avadosdk build --provider http://80.208.229.228:5001
 
 # http://go.ava.do/install/<IPFS HASH>
 
-# Undo changes to Avado Dockerfile
-sed -e "s/${environment_id}/master-goerli/" ./build/Dockerfile > ./build/Dockerfile.tmp && mv ./build/Dockerfile.tmp ./build/Dockerfile 
+# Undo changes restoring backups
+mv ./docker-compose.bak ./docker-compose.yml
```

Comment:
Ideally this is done in a function which is executed via a trap on error and exit, because an error would now leave the state as is.

Reply:
True, added a `cleanup` function